django/applications/catmaid/static/js/widgets/3dviewer.js

```diff
@@ -23,6 +23,12 @@
    * compute with web workers (betweenness centrality shading). */
   var WebGLApplication = function() {
     this.widgetID = this.registerInstance();
+
+    if (CATMAID.MAX_WEBGL_CONTEXTS - CATMAID.countWebGlContexts() < 1) {
+      CATMAID.TooManyWebGlContextsDialog().show();
```

Comment:
You can if you do `(new CATMAID.TooManyWebGlContextsDialog()).show()` (I think you don't even need the parentheses).

Alternatively, you could create a new exception type, that you would throw instead of showing the error dialog directly. Then, `CATMAID.handleError()` could learn how to deal with these (i.e. show the appropriate error dialog). I don't know if this kind of error handling is set up for widgets.